model-optimizer/extensions/middle/ConvertGroupedStridedSlice_test.py

```diff
@@ -987,69 +987,71 @@ def test_ss_shrink_only_long(self):
         (flag, resp) = compare_graphs(graph, graph_ref, 'sslice_2_data', check_op_attrs=True)
         self.assertTrue(flag, resp)
 
-    # test case with 2 strided slices with the same parameters but different outputs
+    # test case with 2 strided slices with the same parameters
```

Comment:
It is really hard to understand what is the test case. Could you provide some comment describing the input graph and the result graph?

Also, we need the following test case:
1. There are 4 SS.
2. 2 of SS slice the same data.
3. 2 others slice the same data.
4. All SS outputs are consumed by different operations.

Reply:
Done.